### PR TITLE
New design > Layout

### PR DESCRIPTION
**Pull request by santiph**
Implement the new design proposed by ken to the Layout in the upgrade flow

### CODE REVIEW

**Comment by Itxaka** [comment]
On the landing page, `SUSE OpenStack Cloud - provided by SUSE` its really close to the `Begin Upgrade` button. Seems like a bit of padding could help there.

Tested the rest, seems good to me, could not find any css issues.

File: assets/app/widgets/suse-footer/suse-footer.less [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/* Start= features/crowbar-footer/crowbar-footer.less */
+suse-footer {
+    color: #5F5F5F;
```
Comment (skazi0) [inline assets/app/widgets/suse-footer/suse-footer.less]: lesslint warning: "hexNotation: #5F5F5F should be written in lowercase."

**Review by skazi0 — CHANGES_REQUESTED** [review]

File: assets/app/widgets/suse-footer/suse-footer.less (hunk 1 above)
Comment (santiph) [reply assets/app/widgets/suse-footer/suse-footer.less]: Good catch! Thanks

**Comment by santiph** [comment]
@Itxaka for the landing page adjustments we have https://trello.com/c/7aSYKiH4/189-3-s51p16-34-landing-page-new-design-frontend-service-integration ;)

**Review by Itxaka — APPROVED** [review]
Great stuff!

**Review by skazi0 — APPROVED** [review]

**Comment by santiph** [comment]
[(2) S51P15 #35 Layout - New Design](https://trello.com/c/e15Q4gsi/188-2-s51p15-35-layout-new-design)

**Comment by skazi0** [comment]
[(?) Update UI Steps - move to New Design](https://trello.com/c/ZBzjl4Ay/218-update-ui-steps-move-to-new-design)

**Pull request merged**